.circleci/config.yml

```diff
@@ -110,13 +112,17 @@ commands:
           corepack enable
           corepack prepare pnpm@10.12.1 --activate
           pnpm build
-          npx tauri build
-          bazel run //:deploy-mac-<<parameters.target-arch>>-dmg -- snapshot
+          rustup target add <<parameters.target-triple>> 
+          npx tauri build --target <<parameters.target-triple>>
+          git rev-parse HEAD > VERSION
+          bazel run --define version=$(git rev-parse HEAD) //:deploy-mac-<<parameters.target-arch>>-dmg -- snapshot
```

Comment:
`git rev-parse HEAD > VERSION` wouldn't be needed, but i can't get `--define version=$(git rev-parse HEAD)` to work for whatever reason.

Reply:
In that case presumably we don't need --define version=$(git rev-parse HEAD) here?